pwa/packages/spwa-rma/src/plugin/component/CmsBlock/CmsBlockContainer.plugin.js

```diff
@@ -0,0 +1,49 @@
+import CmsBlockQuery from 'Query/CmsBlock.query';
+
+// this function injects the new link inside the text content.
+const injectReturnsLinkIntoFooterLinks = (item) => {
```

Comment:
why you just can't plugin in the footer and add your link in there after or before footer links block